core/common/privdata/membershipinfo_test.go

```diff
@@ -23,18 +23,25 @@ func TestMembershipInfoProvider(t *testing.T) {
 		Signature: []byte{1, 2, 3},
 		Data:      []byte{4, 5, 6},
 	}
+	emptyPeerSelfSignedData := protoutil.SignedData{}
 
 	identityDeserializer := func(chainID string) msp.IdentityDeserializer {
 		return &mockDeserializer{}
 	}
 
-	// verify membership provider returns true
-	membershipProvider := NewMembershipInfoProvider(mspID, peerSelfSignedData, identityDeserializer)
+	// verify membership provider pass simple check returns true
+	membershipProvider := NewMembershipInfoProvider(mspID, emptyPeerSelfSignedData, identityDeserializer)
 	res, err := membershipProvider.AmMemberOf("test1", getAccessPolicy([]string{"peer0", "peer1"}))
 	assert.True(t, res)
 	assert.Nil(t, err)
 
-	// verify membership provider returns false
+	// verify membership provider fall back to default access policy evaluation returns true
+	membershipProvider = NewMembershipInfoProvider(mspID, peerSelfSignedData, identityDeserializer)
```

Comment:
I'm not sure if this will actually fall back to the default access policy since the mspID (`peer0` in this case) should satisfy the member orgs of the policy on line 40. You may be able to just remove this specific case since you already test further below when passing an empty mspID it falls back and evaluates to true.

Reply:
Remove this case. Thanks.